app/src/test/java/gov/va/vro/controller/VroControllerTest.java

```diff
@@ -106,7 +105,7 @@ void postFullHealthAssessment() throws Exception {
     request.setDiagnosticCode("7101");
 
     var responseEntity1 =
-        post("/v2/health-data-assessment", request, FullHealthDataAssessmentResponse.class);
+        post("/v1/full-health-data-assessment", request, FullHealthDataAssessmentResponse.class);
```

Comment:
Yes, so, this test while using some mocked items, tests the a v1 path. Previously because we didnt set id type to anything but default the v1 and v2 paths behaved identically so the test could have been argued to cover both (wrongly)

Once we corrected the id types to be correct for v1 vs v2 paths, this started to matter in the asserts. I actually missed a problem during development because I thought the test was fine and didn't discover it until I manually tested endpoints and checked the database. Once I fixed the problem I updated the strings so the test would be more clear what path it's going through.